modules/core/src/main/java/org/apache/ignite/internal/processors/rest/handlers/log/GridLogCommandHandler.java

```diff
@@ -119,7 +119,7 @@ public GridLogCommandHandler(GridKernalContext ctx) {
                         else
                             logFile = new File(req0.path());
                     }
-                    else if (req0.path().startsWith(ctx.config().getIgniteHome()))
+                    else if (req0.path().startsWith(ctx.config().getIgniteHome()) && !req0.path().contains(".."))
```

Comment:
Does this prevent absolute paths and all other vulnerabilities described in https://owasp.org/www-community/attacks/Path_Traversal ?